content/docs/user-guide/data-management/remote-storage/ssh.md

```diff
@@ -13,8 +13,8 @@ with a remote computer, which lets you safely transfer files to and from it
 (like [`scp`]), among other features. Other operations can be used on top of
 SSH, like FTP (simple file transfer protocol) which becomes secure or [SFTP].
 
-[SSH]: https://www.ssh.com/academy/ssh
-[SFTP]: https://www.ssh.com/academy/ssh/sftp-ssh-file-transfer-protocol
+[ssh]: https://www.ssh.com/academy/ssh
```

Comment:
Q: is it done by a linter?

Reply:
Yes, if I run `yarn format content/docs/...` it converts all of these link entries to lowercase